docs/README.md

```diff
@@ -2,8 +2,6 @@
 
 * [Read Me](/README.md)
 * [Introduction](/docs/introduction/README.md)
-  * [Motivation](/docs/introduction/Motivation.md)
```

Comment:
They are not removed - but contents are moved to 'introduction' page - right now it's empty.